Makefile

```diff
@@ -18,7 +18,10 @@ TSDB_BENCHMARK_NUM_METRICS ?= 1000
 TSDB_BENCHMARK_DATASET ?= "$(TSDB_PROJECT_DIR)/testdata/20kseries.json"
 TSDB_BENCHMARK_OUTPUT_DIR ?= "$(TSDB_CLI_DIR)/benchout"
 
-STATICCHECK_IGNORE =
+STATICCHECK_IGNORE = \
+	github.com/prometheus/tsdb/head.go:SA6002 \
```

Comment:
Worth a comment why?

Reply:
We don't elsewhere.

Reply:
yeah I think a comment is useful as I wasted 1 hour  on this one.

Reply:
added